packages/ui/react-ui-form/src/components/ViewEditor/ViewEditor.tsx

```diff
@@ -48,7 +48,7 @@ export const ViewEditor = ({
   schema,
   view,
   registry,
-  readonly,
+  readonly: _readonly,
```

Comment:
I’ve been using  e.g. `propsReadonly` in cases like this, since prefixing with an underscore usually indicates you are discarding or otherwise not using a variable but need to declare it.

Reply:
noted. i've been doing this consistently across 100s of files, so it's an aspect we should discuss and standardize on.